src/index.js

```diff
@@ -554,7 +554,17 @@ class EliteDangerousJournalServer {
     if (event === 'change' && filepath === this.currentJournal) {
       this.getJournalUpdate();
     } else {
-      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occured for`)} ${chalk.magenta(filepath)}`);
+      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occurred for`)} ${chalk.magenta(filepath)}`);
+      const baseName = path.basename(filepath).toLowerCase();
```

Comment:
Could we move the `status.json` logic into a class method like the way `this.getJournalUpdate()` works?

Something like `this.getStatusUpdate()`?

Reply:
The journal updates are new events appended at the end. All the other `*.json` files however are fully rewritten, whenever their content changes. So I would suggest a method that takes the filename and emits an event with such a name: `this.getFileUpdate(filepath)` or maybe (in case the name would conflict with an existing event) `this.getFileUpdate(filepath, eventname)`